### PR TITLE
New EasyConfigs for the Cray programming environment release 2017.08

### PR DESCRIPTION
**Pull request by valtandor**
Including EasyConfigs for the CrayCCE, CrayGNU and CrayIntel variants. We don't have PGI in our version of the Cray programming environment, so someone else would have to do that compiler.

### CODE REVIEW

File: easybuild/easyconfigs/c/CrayCCE/CrayCCE-2017.08.eb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+easyblock = 'CrayToolchain'
+
+name = 'CrayCCE'
+version = '2017.08'
+
+homepage = 'https://pubs.cray.com/browse/xc/article/released-cray-xc-programming-environments-1708---august-3-2017'
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/c/CrayCCE/CrayCCE-2017.08.eb]: trailing white space

File: easybuild/easyconfigs/c/CrayGNU/CrayGNU-2017.08.eb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+easyblock = 'CrayToolchain'
+
+name = 'CrayGNU'
+version = '2017.08'
+
+homepage = 'https://pubs.cray.com/browse/xc/article/released-cray-xc-programming-environments-1708---august-3-2017'
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/c/CrayGNU/CrayGNU-2017.08.eb]: same here

File: easybuild/easyconfigs/c/CrayIntel/CrayIntel-2017.08.eb [hunk 3]
```diff
@@ -0,0 +1,19 @@
+easyblock = 'CrayToolchain'
+
+name = 'CrayIntel'
+version = '2017.08'
+
+homepage = 'https://pubs.cray.com/browse/xc/article/released-cray-xc-programming-environments-1708---august-3-2017'
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/c/CrayIntel/CrayIntel-2017.08.eb]: same here

**Comment by boegel** [comment]
This is quite old now, and we added recent easyconfigs for the Cray toolchains in #8944, so closing this...

Sorry for never getting back to this @valtandor

**Pull request closed without merging**